### PR TITLE
Minor tweaks following a read through

### PR DESCRIPTION
**Pull request by joshuagl**
I spotted a couple of typos while reading through the white paper today.

### CODE REVIEW

File: uptane-whitepaper.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ The second change made to the basic TUF design has to do with the way Uptane ver
...
 
 
-The diagram above illustrates how the checks and balances of the Uptane system works. The connected components on the right hand side of the diagram are on the vehicle, while the components on the left hand-side represent the repositories. The Image repository can be thought of as an authoritative source of information about software images, and the keeper of every image currently deployed by the OEM or supplier, along with the metadata files that prove their authenticity. The Director repository determines what software updates should be distributed to each ECU. 
+The diagram above illustrates how the checks and balances of the Uptane system works. The connected components on the right hand side of the diagram are on the vehicle, while the components on the left hand side represent the repositories. The Image repository can be thought of as an authoritative source of information about software images, and the keeper of every image currently deployed by the OEM or supplier, along with the metadata files that prove their authenticity. The Director repository determines what software updates should be distributed to each ECU.
```
Comment (joshuagl) [inline uptane-whitepaper.md:74]: this hunk just makes hand side vs. hand-side more consistent.

File: uptane-whitepaper.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ The second change made to the basic TUF design has to do with the way Uptane ver
...
+In the first step in the software update process, the vehicle sends its vehicle version manifest for all installed ECUs to the Director repository. This vehicle version manifest contains signed information about existing, installed software images. Using this input, the Director chooses which software images should be installed next. The metadata and software images are distributed to the vehicle, which then runs a verification process. The diagram above shows a Primary ECU that is connected to a number of Secondary ECUs. The Primary ECU downloads software images and metadata from the Director and Image repositories, and then shares them with one or more Secondary ECUs on the same vehicle. ECUs are classified in terms of their access to storage space, memory, a power supply, and (optionally) a direct Internet connection. The chosen form of verification--*full* or *partial*--is based on the available resources of the ECU, as well as the criticality of the software update (for safety and/or security). If the verification process finds no issues, then the software image can be flashed to the ECU, and the vehicle version manifest is updated.
```
Comment (joshuagl) [inline uptane-whitepaper.md:76]: this hunk replaces an emdash (–) with two dashes (--) which seems to increase consistency of the formatting.

File: uptane-whitepaper.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Yet, ignoring software updates is not a viable option either. As Uptane steering
...
... Established OEMs are recognizing the potential of this strategy. In May 2019, GM [announced](https://www.theverge.com/2019/5/21/18633000/gm-ota-software-updates-digital-platform-reuss) that the 2020 Cadillac CT5 sedan would feature an over-the-air software update mechanism, and that such systems will be standard on all GM vehicles in the next four years [9]. In the same year, Ford’s website announced that, beginning in 2020, it would begin equipping most redesigned vehicles in the U.S. with advanced over-the-air software update mechanisms, starting with the all-electric [Mustang Mach-E](https://media.ford.com/content/fordmedia/fna/us/en/news/2020/05/12/new-ford-over-the-air-updates-mustang-mach-e.html) [10]. Moreover, in 2021, [Toyota and Nissan](https://asia.nikkei.com/Business/Automobiles/Toyota-and-Nissan-to-upgrade-driving-functions-remotely) also begin rolling out OTA software update features, Toyota in the new Teammate driver assistance system for the Lexus LS, which will allow for hands-off driving on the highway, and Nissan in additions and adjustments to driving modes that improve the responsiveness of the steering wheel or reduce power consumption in the 2021 Ariya [11].
```
Comment (joshuagl) [inline uptane-whitepaper.md:27]: This hunk just removes some trailing whitespace and changes "for hands-off driving on the highway. and Nissan" -> "for hands-off driving on the highway, and Nissan"

File: uptane-whitepaper.md (hunk 1 above)
Comment (jhdalek55) [reply uptane-whitepaper.md:74]: fine.

File: uptane-whitepaper.md (hunk 3 above)
Comment (jhdalek55) [reply uptane-whitepaper.md:27]: Thanks for taking such a close look. These changes are all fine.

**Pull request merged**